docs/create_robots_whoami.md

```diff
@@ -33,16 +38,14 @@ You can test your new `panda_whoami` package by calling `rai_whoami` services:
 
 2. Building and sourcing the install
```

Comment:
I think the title is not a little bit off

Reply:
https://github.com/RobotecAI/rai/pull/239/commits/44392f0fe3ac1ca494f87beb0f5523b1e3c7b805